tests/docker_service_fixtures.py

```diff
@@ -110,7 +110,7 @@ def stop(self, name: str) -> None:
 
     def down(self) -> None:
         if not SKIP_DOCKER_COMPOSE:
-            self.run_command("down", "--remove-orphans", "--volumes", "-t", "10")
+            self.run_command("down", "-t", "10")
```

Comment:
Won't this potentially cause problems on subsequent runs locally when `--remove-orphans` isn't used?

Reply:
I didn't want to remove that, but I did as a test to see if it fixes the issues I'm seeing. 

The tests just appear to hang around 88% in CI and around 92% when I run it locally. 

I'll add it back as I don't think it's causing this issue.